### ISSUE TITLE
Fix malawi FCT loading

### ISSUE THREAD

**Issue by rbroth** [micronutrientsupport/database-architecture#233]
File `test-data\V0.3.0.10__malawi-data.sql` insert food items for Malawi. The equivalent food composition table csv is `Food Composition Tables\CSV\MAPS_MAFOODS_v1.4.csv`. The SQL script also insert data into the citation table (citations for the individual food item entries). When the sql script is removed, no rows are inserted into the citation table; but when the sql script is kept, duplicate rows are inserted into the fooditem table.

The lack of citations being inserted may be due to lack of data being provided in csv format.

We want to fix the MAFOODS food importing script to also import citation data

**Comment by rbroth** [comment]
@spenny-liam @LuciaSegovia

**Comment by LuciaSegovia** [comment]
I don't really understand what you mean, but since you're working with the FCTs... I had a script that does all the Malawi FCT cleaning ([here](https://github.com/LuciaSegovia/MAPS_fct/blob/main/mafood.R)) maybe we could intergrate that so if I update the FCT then I could just share it. I could even try to save it as sql ... 🤓

**Comment by spenny-liam** [comment]
@LuciaSegovia Before we set up the standard format and started loading your files directly, Roman had a citation for each individual food item. Here's an example of the citation information for each food item:
![image](https://user-images.githubusercontent.com/76099100/132633407-8d1dcfe5-26ad-425c-bcbc-e69c98c2d416.png)

**Comment by spenny-liam** [comment]
> I don't really understand what you mean, but since you're working with the FCTs... I had a script that does all the Malawi FCT cleaning ([here](https://github.com/LuciaSegovia/MAPS_fct/blob/main/mafood.R)) maybe we could intergrate that so if I update the FCT then I could just share it. I could even try to save it as sql ... 🤓

I dont quite understand what you want to do. But if it is changes to the MAFOODS data. Could you just update the file on teams MAPS_MAFOODS.csv? We have python scripts that automatically load any of those standard format files into the database

**Comment by rbroth** [comment]
The problem is that the malawi FCT has citations for individual food items. We want to import those citations. Previously, I had written a script in SQL that imports both the malawi FCT food items as well as their citations. 

Now, we want to move to a consistent system, where we import all FCTs via csv files and the python scripts. 

@LuciaSegovia as far as I can tell, none of the FCT csv files contain citation data. Could you add a `citation` column to the csvs that contains the citation for that fooditem?

@spenny-liam do the python scripts have a facility for importing citation data? If not, could you open an issue (low priority) to add that functionality, and work with Lucia on determining the format for the csv file.

**Comment by LuciaSegovia** [comment]
@rbroth you mean the citation_id? I have now the "data_reference_original_id". Is that the same?

**Comment by rbroth** [comment]
Yes, I believe the citation_id. I also just realised that it's a misleadingly-named column

**Comment by LuciaSegovia** [comment]
Do you still need the full citation? It will be in a different table because for Kenya and West-Africa, they can have multiple citations per food item. Is that ok?
